src/services/donationService.ts

```diff
@@ -75,6 +75,26 @@ export async function fetchDonationsInfo():
 }
 
 
+export async function unsetPendingAmountRemainingFromCommittedDonations(options:{
+  report :ReportInterface
+}){
+  const {report} = options;
+  const query =  {status :DonationStatus.COMMITTED,
```

Comment:
It's not necessary to be COMMITTED, it should just not be PENDING

Reply:
@aminlatifi  You fixed it in `develop` branch

https://github.com/Giveth/giveth-1-simulation/blob/3f10671e84b3c4ea11c4d5b4bbe7091d505eea30/src/services/donationService.ts#L82-L84